.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,142 @@
+name: Build and test
+
+on:
+  pull_request:
+    branches:
+      - main
+      - develop
+  push:
+    branches:
+      - develop
+      - 'wheel/**'
+  release:
+    types:
+      - created
+      - edited
+
+jobs:
+  aqt-checks:
+    name: AQT - Build and test module
+    strategy:
+      matrix:
+        os: ['ubuntu-20.04', 'macos-11', 'windows-2019']
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: '0'
+    - run: git fetch --depth=1 origin +refs/tags/*:refs/tags/* +refs/heads/*:refs/remotes/origin/*
+    - name: Set up Python 3.8
+      if: github.event_name == 'push'
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.8'
+    - name: Build and test (3.8)
+      if: github.event_name == 'push'
+      run: |
+        ./.github/workflows/build-test nomypy
+    - name: Set up Python 3.9
+      if: github.event_name == 'pull_request' || github.event_name == 'release' || contains(github.ref, 'refs/heads/wheel')
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.9'
+    - name: Build and test including remote checks (3.9) mypy
+      if:  (matrix.os == 'macos-11') && ((github.event_name == 'pull_request' &&  github.event.pull_request.head.repo.full_name == github.repository) || github.event_name == 'release' || contains(github.ref, 'refs/heads/wheel'))
+      run: |
+        ./.github/workflows/build-test mypy
+    - name: Build and test including remote checks (3.9) nomypy
+      if:  (matrix.os != 'macos-11') && ((github.event_name == 'pull_request' &&  github.event.pull_request.head.repo.full_name == github.repository) || github.event_name == 'release' || contains(github.ref, 'refs/heads/wheel'))    
+      run: |
+        ./.github/workflows/build-test nomypy
+    - name: Set up Python 3.10
+      if: github.event_name == 'push' || github.event_name == 'pull_request'
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.10'
+    - name: Build and test (3.10)
+      if: github.event_name == 'push' || github.event_name == 'pull_request'
+      run: |
+        ./.github/workflows/build-test nomypy
```

Comment:
Tests are not currently running on Windows. These calls should have `shell: bash` in the workflow to fix this I think.

Reply:
Thank you, will try that!